setup.py

```diff
@@ -178,14 +164,6 @@
                 shutil.copy2(path_gcc_lib,os.path.join(".","src","pyfmi"))
                 gcc_lib = os.path.join(".","src","pyfmi","libgcc_s_dw2-1.dll")
 
-if no_msvcr:
```

Comment:
I'm a little bit hesitant to remove this for now as I'm not sure what the consequences are (or if this is something old and unused). Is it possible to change this for now to make sure we still support the `no_msvcr` if Python version is less than 3.12?
Other than this I think the changes look go od. I would just add a changelog entry.

Reply:
It looks that is only needed if mingw is used, do you still plan on using it ?
We dont need that for conda-forge.

Reply:
It is still necessary for users of PyFMI on Windows, hence why we are not able to drop it at the moment.

Reply:
ok